### ISSUE TITLE
Test failures on s390x

### ISSUE THREAD

**Issue by cuviper** [srijs/rust-crc32fast#5]
I just tried to build this package for Fedora, and two tests failed on s390x:
https://koji.fedoraproject.org/koji/taskinfo?taskID=31253677

```text
failures:
---- baseline::test::fast_16_is_the_same_as_slow stdout ----
thread 'baseline::test::fast_16_is_the_same_as_slow' panicked at '[quickcheck] TEST FAILED. Arguments: (0, [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0])', /usr/share/cargo/registry/quickcheck-0.6.2/src/tester.rs:179:28
note: Run with `RUST_BACKTRACE=1` for a backtrace.
---- test::combine stdout ----
thread 'test::combine' panicked at '[quickcheck] TEST FAILED. Arguments: ([0], [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0])', /usr/share/cargo/registry/quickcheck-0.6.2/src/tester.rs:179:28
```

I haven't investigated yet, but given that all other arches passed, I'd first suspect a big-endian issue.

**Comment by cuviper** [comment]
I tried that with a [new build](https://koji.fedoraproject.org/koji/taskinfo?taskID=31256848), and it passed, thanks!

**Comment by srijs** [comment]
Great! I'm just in the process of setting up CI for big endian via `cargo cross`, once I have that done I'll go and cut a fix release with this.

**Comment by srijs** [comment]
Published as `1.1.2`.
